### PR TITLE
Null Safety and Example App

### PR DESCRIPTION
**Pull request by bradintheusa**
This has null safety and I made the example app runnable.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 name: easy_dialog
 description: Easy Dialog package helps you easily create basic or custom dialogs.
-version: 1.0.5
+version: 2.0.0
```
Comment (JavierDuarteC) [inline pubspec.yaml]: Major versión number is changed only when new features are added, you could update it to 1.1.0 instead

**Review by JavierDuarteC — CHANGES_REQUESTED** [review]

**Comment by bradintheusa** [comment]
Change applied. Version is now 1.1.0

File: pubspec.yaml (hunk 1 above)
Comment (bradintheusa) [reply pubspec.yaml]: Agreed. I got a bit too ambitious there.

**Review by JavierDuarteC — APPROVED** [review]
Thanks a lot!

**Pull request merged**